lib/ansible/modules/cloud/ovirt/ovirt_vm.py

```diff
@@ -2488,7 +2488,8 @@ def kernel_persist_check():
                         wait_condition=lambda vm: vm.status == otypes.VmStatus.UP,
                     )
             # Allow migrate vm when state present.
-            vms_module._migrate_vm(vm)
+            if vm:
```

Comment:
Don't we need ```if vm``` also before line 2483 which contains ```if vm.next_run_configuration_exists:```

Reply:
I think not, you get `vm` var on line 2482 with `ret['id']` and virtual machine is always defined at that moment.